### PR TITLE
Improvements to the InputEditor

### PR DESCRIPTION
**Pull request by Robbware**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-WEB [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-WEB-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description

The following changes will allow us to show/hide the Name and Short name fields on the InputEditor, via a configuration file. Additionally, a MultiComboBox component was created as DevExpress, by default, does not have a dropdown component which enables multiple choice.

### CODE REVIEW

**Comment by Robbware** [comment]
Fixing the failing tests.

File: .gitignore [hunk 1]
```diff
@@ -349,3 +349,6 @@ MigrationBackup/
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
 /COMETwebapp/Tools
+
+# Jetbrains folders
+.idea/
```
Comment (jaimeatrhea) [inline .gitignore:354]: Please add a new line at the end of the file

File: COMET.Web.Common/Components/MultiComboBox.razor.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
+﻿// -----------------------------------------------------------------------------------
+//   <copyright file="MultiComboBox.razor.cs" company="RHEA System S.A.">
+//      Copyright (c) 2023 RHEA System S.A.
+// 
+//      Authors: Sam Gerené, Jaime Bernar
```
Comment (jaimeatrhea) [inline COMET.Web.Common/Components/MultiComboBox.razor.cs:5]: This component was copied from another project called AIDA. Please align the headers with the headers for the WebApp

File: COMET.Web.Common/Components/MultiComboBox.razor.css
Comment (jaimeatrhea) [inline COMET.Web.Common/Components/MultiComboBox.razor.css:1]: This css file as is in the COMET.Web.Common is not going to be loaded. This styles need to be in the main app styles file (if they are already there just remove this file)

**Review by jaimeatrhea — COMMENTED** [review]
Some small requests

**Review by samatrhea — CHANGES_REQUESTED** [review]

**Comment by Robbware** [comment]
Working on the coverage, just a bit more 😉

**Review by samatrhea — APPROVED** [review]

**Review by jaimeatrhea — APPROVED** [review]

**Pull request merged**